lib/commands/binlog_dump.js

```diff
@@ -3,6 +3,44 @@ var util = require('util');
 var CommandCode = require('../constants/commands');
 var Packets = require('../packets');
 
+var MysqlTypes =  {
```

Comment:
this duplicates https://github.com/sidorares/node-mysql2/blob/master/lib/constants/types.js

Reply:
Great. I will use that one.